Tests/BackendIntegrationTests/SignatureVerificationIntegrationTests.swift

```diff
@@ -335,6 +341,23 @@ class DynamicModeSignatureVerificationIntegrationTests: BaseSignatureVerificatio
 
 }
 
+/// Header verification (see `HTTPRequest.headerParametersForSignatureHeader`) is enabled by default,
+/// but this helps verify that the backend is still signing correctly without it for older SDK versions.
+class SignatureVerificationWithoutHeaderHashIntegrationTests: EnforcedSignatureVerificationIntegrationTests {
+
+    override var disableHeaderSignatureVerification: Bool { return true }
```

Comment:
This allows us to run all of `EnforcedSignatureVerificationIntegrationTests` without the new feature, so we can verify that the older SDKs continue to work.